src/features/advanced/exp-expert-eta.ts

```
@@ -0,0 +1,142 @@
+  'ELECTRONICS',
+  'FOOD_INDUSTRIES',
+  'FUEL_REFINING',
+  'MANUFACTURING',
+  'METALLURGY',
+  'RESOURCE_EXTRACTION',
+];
+
+const expertDays = [10.0, 12.5, 57.57, 276.5, 915.1];
+
+function onTileReady(tile: PrunTile) {
+  const site = sitesStore.getById(tile.parameter)!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)